frontend/src/pages/entities/EntityAnalysisPage.tsx

```diff
@@ -75,7 +82,33 @@ const EntityAnalysisPage = () => {
   const [isLoading, setIsLoading] = useState(true);
   const [apiError, setApiError] = useState<ApiError>();
 
+  const tryToCreateVideo = async () => {
+    if (pollName !== YOUTUBE_POLL_NAME) {
+      return false;
+    }
+    if (!isLoggedIn) {
+      return false;
+    }
+    const videoId = extractVideoId(uid);
+    if (!videoId) {
+      return false;
+    }
+    try {
+      await VideoService.videoCreate({
```

Comment:
Related to #1867. It could be easily forgotten. Commenting only so that it gets mentioned on the ticket.